CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## vNext
 
-- ...
+- Update config, chain-spec file to enable Cere native to Cere ERC20 transfer
```

Comment:
Just use `Enable CERE native to CERE ERC20 transfers`. It is clear from the diff what has been changed.

Reply:
Updated the changelog